bdd/jenkins/ci.sh

```diff
@@ -164,7 +164,7 @@ function install_tools() {
 # Creates the GKE test cluster
 ###############################################################################
 function create_cluster() {
-	CLUSTER_NAME=$(echo ${BRANCH_NAME,,}-${BUILD_ID,,}-lh-tekton-e2e)
+	CLUSTER_NAME=$(echo ${BRANCH_NAME,,}-${BUILD_ID,,}-lh-jenkins-e2e)
```

Comment:
💯  good catch. I have seen this issue myself, but could not figure out why I sometimes would see a failure. That's what I get for starting off with an existing script and not carefully doing a find&replace first :-)